common_utils/shareMem.go

```diff
@@ -0,0 +1,58 @@
+package common_utils
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	memKey = 7749
+	memSize = 1024
```

Comment:
>memSize [](http://example.com/codeflow?start=1&length=7)

How do you figure out the size? In what unit?

Reply:
updated